pkg/handler/common/cluster.go

```diff
@@ -246,7 +246,7 @@ func GenerateCluster(ctx context.Context, projectID string, body apiv1.CreateClu
 	return partialCluster, nil
 }
 
-func GetExternalClusters(ctx context.Context, userInfoGetter provider.UserInfoGetter, clusterProvider provider.ClusterProvider, projectProvider provider.ProjectProvider, privilegedProjectProvider provider.PrivilegedProjectProvider, seedsGetter provider.SeedsGetter, projectID string) ([]*apiv1.Cluster, error) {
+func GetClusters(ctx context.Context, userInfoGetter provider.UserInfoGetter, clusterProvider provider.ClusterProvider, projectProvider provider.ProjectProvider, privilegedProjectProvider provider.PrivilegedProjectProvider, seedsGetter provider.SeedsGetter, projectID string) ([]*apiv1.Cluster, error) {
```

Comment:
The name was misleading before as we have introduced the external clusters functionality. In this case external was referring to the externally exposed API type.